fedimint-server/src/consensus/server.rs

```diff
@@ -838,7 +845,13 @@ async fn submit_module_consensus_items(
                     }
                 }
 
-                sleep(Duration::from_secs(1)).await;
+                drop(dbtx);
+
+                tokio::select! {
+                    biased;
+                    _ = sleep(Duration::from_secs(1)) => {},
```

Comment:
nit: Should likely use a tokio [`Interval`](https://docs.rs/tokio/latest/tokio/time/struct.Interval.html) so this doesn't reset.

Reply:
resetting after submitting consensus proposal feels like correct behavior?

Reply:
Only if all are polled, otherwise one module might starve others.